React/Views/RCTWrapperViewController.m

```diff
@@ -133,4 +140,18 @@ - (void)didMoveToParentViewController:(UIViewController *)parent
   }
 }
 
+- (UIImageView *)findHairlineImageViewUnder:(UIView *)view {
+    if ([view isKindOfClass:UIImageView.class] && view.bounds.size.height <= 1.0) {
```

Comment:
This isn't the worst thing in the world, but there isn't a better way to grab the shadow?